integration/src/test/java/org/apache/iotdb/db/integration/aligned/IoTDBEmptyDataIT.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.integration.aligned;
+
+import org.apache.iotdb.db.utils.EnvironmentUtils;
+import org.apache.iotdb.jdbc.Config;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class IoTDBEmptyDataIT {
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    EnvironmentUtils.envSetUp();
+    Class.forName(Config.JDBC_DRIVER_NAME);
```

Comment:
Please reference `IoTDBSimpleQueryIT` to adjust our cluster IT framework

Reply:
fix

Reply:
Adjust IT framework hints:
1. Remove all `Class.forName`, and add `ClusterTest.class` to the category, as this is a pure black-box test.
2. Use `EnvFactory.getEnv().initBeforeClass();` to init the environment in a class level
3. Use `EnvFactory.getEnv().cleanAfterClass();` to clean the environment in a class level
4. Use the following codes to get connection
```java
try (Connection connection = EnvFactory.getEnv().getConnection();
        Statement statement = connection.createStatement()) {
  ...
}
```